pyspark/test/bigdl/keras1/test_layer.py

```diff
@@ -55,7 +53,11 @@ def __dump_keras(self, keras_model, dump_weights=False):
         return keras_model_json_path, keras_model_hdf5_path
 
     def __generate_model(self, input_data, output_layer):
-        input_node = [Input(shape=input_data.shape[1:]) for i in input_data] # it's a list incase of Merge Layer
+        import keras
```

Comment:
move `import keras` to the header of this file

Reply:
DONE